server/sonar-server/src/main/java/org/sonar/server/platform/ws/UpgradesAction.java

```diff
@@ -70,7 +70,7 @@ private static void writeMetadata(JsonWriter jsonWriter, Release release) {
 
   @Override
   public void define(WebService.NewController controller) {
-    controller.createAction("upgrades")
+    controller.createAction(ARRAY_UPGRADES)
```

Comment:
The fact two constants have the same value doesn't necessary means we have to use the same constant. Here `ARRAY_UPGRADES` is an element of the JSON response. The fact it has the same value is `coincidental`, and using the same constant make the code less readable by giving the impression the intent was to use the same value in the two places.